assets/blocks/course-completed-actions/block.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "sensei-lms/course-completed-actions",
+  "category": "sensei-lms",
+  "textdomain": "sensei-lms",
```

Comment:
TIL =)

Reply:
~~With this, would we be able to move title, description and keywords to the block.json? Or is it something new and it wouldn't work in WP `5.6`?~~ Just confirmed that it's really new :)